lib/modules/ai_bot/commands/setting_context_command.rb

```diff
@@ -0,0 +1,148 @@
+#frozen_string_literal: true
+
+module DiscourseAi::AiBot::Commands
+  MAX_CONTEXT_TOKENS = 2000
+
+  class SettingContextCommand < Command
+    def self.rg_installed?
+      @rg_installed ||=
+        begin
+          Discourse::Utils.execute_command("which", "rg")
+          true
+        rescue Discourse::Utils::CommandError
+          false
+        end
+    end
+
+    class << self
+      def name
+        "setting_context"
+      end
+
+      def desc
+        "Will provide you with full context regarding a particular site setting in Discourse"
```

Comment:
Should this (and description below) be localized with I18n?

Reply:
its a very very tough one, this is instructions for the LLM, not humans, interacting with LLMs not in English is not super strong at the moment when it comes to mega fancy stuff.

Reply:
Ahh right I didn’t realise it went to the LLM, nvm!